torch/_inductor/codecache.py

```diff
@@ -1217,6 +1228,17 @@ def pick_vec_isa() -> VecISA:
     if not _valid_vec_isa_list:
         return invalid_vec_isa
 
+    # Set simdlen according to _get_cpu_capability to allow using
+    # the environment ATEN_CPU_CAPABILITY to control CPU vec ISA
+    if (
+        config.cpp.simdlen is None
+        and not isinstance(_valid_vec_isa_list[0], VecNEON)
+        and not isinstance(_valid_vec_isa_list[0], VecZVECTOR)
+    ):
+        config.cpp.simdlen = get_simdlen_from_cpu_capability(
+            torch._C._get_cpu_capability()
+        )
+
     # If the simdlen is None, it indicates determin the vectorization length automatically
     if config.cpp.simdlen is None:
         assert _valid_vec_isa_list
```

Comment:
perhaps we don't need this check any longer.

Reply:
Removed the check